db_upgrade_scripts/mosip_pms/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql

```diff
@@ -17,6 +17,7 @@ GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA pms TO postgres;
 \ir ../ddl/pms-secure_biometric_interface.sql
 \ir ../ddl/pms-secure_biometric_interface_h.sql
 \ir ../ddl/pms-device_detail_sbi.sql
+\ir ../ddl/pms-oidc_client.sql
```

Comment:
I believe this table was introduce in B2 release, so this should be part of B1 to B2 upgrade sql file